packages/affine/data-view/tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
+  "extends": "../../../tsconfig.json",
+  "compilerOptions": {
+    "rootDir": "./src/",
+    "outDir": "./dist/",
+    "noEmit": false
+  },
+  "include": ["./src"],
+  "references": []
```

Comment:
You need to set `reference` according to your dependencies like https://github.com/toeverything/blocksuite/blob/master/packages/affine/block-list/tsconfig.json#L9

In a new project, this will make sure the dependencies built before ts build this package.